tasks/main.yml

```
@@ -2,6 +2,22 @@
 - name: Set version specific variables
   include_tasks: set_vars.yml
 
+- name: Scan sudoers
+  scan_sudoers:
+    output_parsed_configs: true
+    output_raw_configs: false
+#  no_log: true
+#  when: sudo_check_if_configured | bool
+
+- name: Sudo is already configured - skip rest of role
+  ansible.builtin.meta: end_host
+  when:
+    - __actual == sudo_sudoers_files
+    - sudo_check_if_configured | bool
+  vars:
+    __actual: "{{ ansible_facts['sudoers'].sudoers_files | map('dict2items') |
+      map('rejectattr', 'key', 'match', '^include_files$') | map('items2dict') }}"
```

```suggestion
      map('rejectattr', 'key', 'match', '^include_files$') | map('list') |
      map('items2dict') | list }}"
```
This should fix the breakage with ansible 2.9, which requires every filter pipeline like this to be "cast" to a `list`.